backend/clickhouse/logs.go

```diff
@@ -272,13 +273,30 @@ func (client *Client) ReadLogsHistogram(ctx context.Context, projectID int, para
 
 func (client *Client) LogsKeys(ctx context.Context, projectID int, startDate time.Time, endDate time.Time) ([]*modelInputs.LogKey, error) {
 	sb := sqlbuilder.NewSelectBuilder()
-	sb.Select("arrayJoin(LogAttributes.keys) as key, count() as cnt").
-		From(LogsTable).
-		Where(sb.Equal("ProjectId", projectID)).
-		GroupBy("key").
-		OrderBy("cnt DESC").
-		Where(sb.LessEqualThan("toUInt64(toDateTime(Timestamp))", uint64(endDate.Unix()))).
-		Where(sb.GreaterEqualThan("toUInt64(toDateTime(Timestamp))", uint64(startDate.Unix())))
+
+	startOfCurrentHour := time.Now().UTC().Truncate(1 * time.Hour)
+	if endDate.UTC().After(startOfCurrentHour) {
```

Comment:
why do we need a condition here at all if the materialized view is always up to date (there isn't a refresh interval for it, right?)

rounding to the start of the hour seems like would produce a skewed result (earlier times in the hour will have less of a chance to use the materialized view).